style/roads.mss

```diff
@@ -23,12 +24,14 @@
 @helipad-fill: @aeroway-fill;
 @access-marking: #eaeaea;
 @access-marking-living-street: #cccccc;
+@access-marking-busway: #cbbce6;
 
 @default-casing: white;
 @tertiary-casing: #8f8f8f;
 @residential-casing: #bbb;
 @road-casing: @residential-casing;
 @service-casing: @residential-casing;
+@busway-casing: #b099d9;
```

Comment:
There is very high contrast between the fill and the casing color. This casing color `#b099d9` is Lch (67, 36, 306), which is 9 darker than residential casing. The chroma (similar to saturation) of the `@busway-fill:` `#f2eef9` is only 6, and the Lightness is 95, so this is quite a big jump in chroma and darkness, leading to a strong color compared to the fill. Please compare to road-colors-generated.mss